hack/common.sh

```diff
@@ -788,6 +793,87 @@ function os::build::ldflags() {
 }
 readonly -f os::build::ldflags
 
+# os::build::enable_swap attempts to enable swap for the system if a) this is Linux and b)
+# the amount of physical memory is less than 10GB. This is a stopgap until we have
+# better control over memory use in Go 1.7+.
+function os::build::enable_swap() {
```

Comment:
I would happily contribute my memory if it meant a) we didn't need to do this or b) the thought of it wouldn't haunt me